client/src/components/App.js

```diff
@@ -61,29 +62,26 @@ class App extends Component {
     };
 
     const { authed, checkingAuth } = this.state;
-    console.log(this.state);
     return (
       <BrowserRouter>
         <Connect authOptions={authOptions}>
           <div className="App">
             <Switch>
-              <Route exact path="/">
-                {!authed ? (
-                  <>{checkingAuth ? <div>LOADING..</div> : <Login />}</>
-                ) : (
-                  <div>
-                    <DiagnosticContainer userSession={userSession} handleSignOut={this.handleSignOut} />
-                  </div>
-                )}
-              </Route>
+              <PrivateRoute
+                exact
+                path="/"
+                authed={authed}
+                component={() => <DiagnosticContainer userSession={userSession} handleSignOut={this.handleSignOut} />}
+              />
+
               {/* ADD/EDIT ROUTES WITH THEIR COMPONENTS HERE: */}
               <Route path="/signup" />
-              <Route path="/symptomsurvey" />
-              <Route path="/log" />
-              <Route path="/healthlog" />
+              <PrivateRoute path="/symptomsurvey" authed={authed} />
+              <PrivateRoute path="/log" authed={authed} />
+              <PrivateRoute path="/healthlog" authed={authed} />
               <Route path="/education" render={() => <FactQuizContainer handleSignOut={this.handleSignOut} />} />
               <Route path="/map" />
```

Comment:
should /education and /map paths be only for logged in users?

Reply:
^ Up for discussion, but I'd say yes, at least for the time being. Otherwise, we'd have to add a way to reach eduction or map before logging in. Which, now that I'm thinking about it, could be a useful feature down the line, but I don't think we should concern ourselves with it now.

Reply:
*updated, /education and /map paths are also PrivateRoutes.